erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+from frappe import _
+
+def get_data():
+	return {
+		'fieldname': 'stock_entry',
+        'non_standard_fieldnames': {
+			'Qulaity Inspection': 'reference_name'
+		},
+		'internal_links': {
+			'Quality Inspection': ['items', 'quality_inspection']
+		},
+        'transactions': [
+			{
+				'label': _('Reference'),
+				'items': ['Quality Inspection']
+			}
+		]
```

Comment:
If we're creating this anyway, should we add other links as well (for example, Work Order)?

Reply:
Material Request and Purchase Receipt link added.

Reply:
The following doctypes are linked to Stock Entry (or found in the items table):

```
Work Order
Purchase Order
Delivery Note
Sales Invoice
BOM
Warehouse
Address
Supplier
Customer
Project
Job Card
Pick List
Batch
Package Tag
```

Talk to Arundhati and Swati, and see if we should pull any of these?

Reply:
Let's make separate tasks for these